.github/workflows/get-changelog-from-changelog-txt/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Get Changelog From changelog and readme Text Files
+description: Get Changelog From changelog.txt and readme.txt Text Files - publish in wordpress.org.
+
+inputs:
+  VERSION:
+    description: 'Package Version'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      env:
+        VERSION: ${{ inputs.VERSION }}
+      run: |
+        npm install --no-package-lock --no-save marked@2.0.6
```

Comment:
This is a very old version of `marked`, why not use the latest version?

Reply:
This is how it's done in the pro repo, I can update this to be the latest version however we only tested this version. Any chance the diffs are critical in that way?

Reply:
I honestly don't know, but you can test it locally
It would be much better to be up to date with our dependencies

Reply:
Done